auth/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+# services:
```

Comment:
Is this commented out part still necessary?

Reply:
No, removed it, thanks!